doc/source/operations/rocky-linux-9.rst

```diff
@@ -388,6 +388,29 @@ The possible batches depend on a number of things:
 * available spare hypervisor capacity
 * sizes of groups of compatible hypervisors
 
+Transition to Containerized Libvirt
```

Comment:
It would be better if this was inserted into the correct location within the existing procedure. We'd at least want to drain the node first.

Reply:
moved it before host configure and added note